src/lib/ecl/attitude_fw/ecl_pitch_controller.cpp

```diff
@@ -96,15 +96,21 @@ float ECL_PitchController::control_attitude(const struct ECL_ControlData &ctl_da
 
 	/* input conditioning */
 	float airspeed = constrain_airspeed(ctl_data.airspeed, ctl_data.airspeed_min, ctl_data.airspeed_max);
-	/* calculate the offset in the rate resulting from rolling  */
-	//xxx needs explanation and conversion to body angular rates or should be removed
-	float turn_offset = fabsf((CONSTANTS_ONE_G / airspeed) *
-				  tanf(roll) * sinf(roll)) * _roll_ff;
```

Comment:
I'm going to be testing this with and without feed forward compensation this week as well.  Even if feed forward compensation is necessary I think _pitch_ff should be applied as I'm guessing people will not think to adjust roll gains after seeing the nose pitch down or bad pitch tracking from log files.